satpy/tests/reader_tests/test_olci_nc.py

```diff
@@ -180,8 +183,10 @@ class TestBitFlags(unittest.TestCase):
 
     def test_bitflags(self):
         """Test the BitFlags class."""
-        import numpy as np
         from functools import reduce
+
+        import numpy as np
+
```

Comment:
Why do we need an empty line after the imports here, but not in line 120/121 or 153/154?

Reply:
It is separating the imports into 3 (there are actually 4 or 5) groups: builtins, third party, and first party (satpy).